woocommerce/woocommerce
# Update customer invoice emails for failed payments

### Changes proposed in this Pull Request:

This updates the email templates for the customer invoice email (aka "Send order details" email) to have different wording at the top when the order has a failed status. The previous wording could be confusing since it asks for payment but doesn't explain that their previous payment attempt failed.

Fixes #52883

### How to test the changes in this Pull Request:

1. Create three orders, one with `pending` status, one with `failed` status, and one with `completed` status.
2. If needed, install a plugin like [WP Mail Logging](https://wordpress.org/plugins/wp-mail-logging/) to capture outgoing emails from WP's mail function.
3. Go to the edit order screen for each of these orders, and from the "Order actions" box in the upper right corner, choose "Send order details to customer" and click the arrow button next to the dropdown.
4. Now check the email log. The `pending` email should contain the same "An order has been created for you" message as before this change. The `failed` email should contain a message starting with "Sorry your order on {site name} was unsuccessful". The `completed` email should also be unchanged, with a "Here are the details of your order" message.
5. Go to WooCommerce > Settings > Emails > Order details. Change the email type to plain text. Try sending each of the three emails again, and ensure that the outgoing emails contain the same messages as before, but in plain text.
